dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/operator/math/BeamSqlSignExpression.java

```diff
@@ -37,7 +37,13 @@ public BeamSqlSignExpression(List<BeamSqlExpression> operands) {
     BeamSqlPrimitive result = null;
     switch (op.getOutputType()) {
```

Comment:
sound right, need to rewrite the constructors for `BeamSqlMathUnaryExpression`, otherwise it fails with nested methods like `SIGN(ABS($1))`.